packages/wxt/src/client/content-scripts/content-script-context.ts

```diff
@@ -163,6 +163,38 @@ export class ContentScriptContext implements AbortController {
     return id;
   }
 
+  /**
+   * Clears a timeout set by ctx.setTimeout.
+   */
+  clearTimeout(id: number | null): void {
+    if (id !== null) {
+      clearTimeout(id);
+    }
+  }
```

Comment:
These should be `| undefined` if we decide to keep them. And we don't need an if-statement either.

```suggestion
  clearTimeout(id: number | undefined): void {
    clearTimeout(id);
  }
```

Same applies to `clearInterval`.